Segment.java

```diff
@@ -0,0 +1,11 @@
+import java.util.concurrent.atomic.*;
+
+public class Segment {
+    private static final int SEGMENT_SIZE = 4;
```

Comment:
Since this is defined also in Segment Table, please create only one constant and pass it into the constructor.